keps/79-hierarchical-cohorts/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: 
+kep-number: 79
+authors:
+  - "@mwielgus"
+status: draft
+creation-date: 2023-12-28
+reviewers:
+  - "@tenzen-y "
+  - "@alculquicondor"
+approvers:
+  - "@alculquicondor"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.7"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: "v0.7"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+disable-supported: true
```

Comment:
Add the feature gate name used to disable the feature if needed.

Reply:
Flipped to true. Api changes should probably be not feature gated.